### ISSUE TITLE
Craft 5 Upgrade: Scraper Plugin Status

### ISSUE THREAD

**Issue by viesrood** [TopShelfCraft/Scraper#9]
Hi Michael,

Do you know if the Scraper plugin will be updated to Craft 5 soon? I’m unable to proceed with the upgrade since this is the only plugin that isn’t ready yet.

Thanks in advance!

**Comment by michaelrog** [comment]
Yeah, I do intend to update Scraper for Craft 5. It's a breaking change under the hood (from `fabpot/goutte` to `symfony/browser-kit`), and I expect it to be out this month.

**Comment by viesrood** [comment]
Thank you for your information. Good luck with the update.

**Comment by afairhurst** [comment]
Hi Michael,

I hope you're well. I'm curious to know if you still plan to upgrade this plugin for Craft 5?

Many thanks,

Andrew

**Comment by viesrood** [comment]
I understand you’re probably busy, but could you share something about the status of this plugin in relation to CraftCMS 5?

**Comment by viesrood** [comment]
Any news about the upgrade of this plugin to Craft 5?
